### PR TITLE
Minor fixes: phony targets, missing shell escaping, missing comments !

### PR DESCRIPTION
**Pull request by maugier**
Also make test code compliant with the variable naming scheme -- sorry for not getting it right from the beginning ! I am a very sloppy coder.

### CODE REVIEW

File: Makefile [hunk 1]
```diff
@@ -85,9 +88,10 @@ $(VARIABLECONTAININGTHETARGETNAME): $(VARIABLECONTAININGTHESOURCEFILECOMPUTEDFRO
 # This rule implements a basic unit test
 # TODO: every line should have its own comment
 test:
+	# recursively descent into the test directory and invoke make
```
Comment (calops) [inline Makefile]: The first letter should be capitalized.
s/descent/descend/

**Comment by maugier** [comment]
Closing and reopening to follow good practice because the contents have changed

**Comment by maugier** [comment]
Reopening to follow good practice because the contents have changed

**Comment by gcmalloc** [comment]
A comment to say that I approve those changes.

**Comment by maugier** [comment]
A comment to say that I approve of this comment

**Pull request closed without merging**